ytools/yast2/create_maintenance_branch

```diff
@@ -87,20 +91,38 @@ end
 
 def modify_dockerfile
   if !File.exist?("Dockerfile")
-    STDERR.puts "No Dockerfile, skipping its adaptation"
+    warn "No Dockerfile, skipping its adaptation"
     return
   end
 
   lines = File.readlines("Dockerfile")
   from_index = lines.index { |l| l =~ /^FROM / }
   raise "Missing FROM in dockerfile" unless from_index
+
   line = lines[from_index]
-  docker_tag = ":" + TARGET.sub(/sp/, "-sp")
-  if line.include?(":") # docker file already use tag
-    line.sub!(/:.*$/, docker_tag + "\n")
+
+  if line.include?("registry.opensuse.org")
+    # new OBS Docker image
+    project_path = TARGET.sub(/sp/, "/sp").sub(/sle/, "sle-")
+
+    registry = if project == "yast"
+      img_type = line.include?("cpp") ? "cpp" : "ruby"
+      "registry.opensuse.org/yast/#{project_path}/containers/yast-#{img_type}"
+    else
+      "registry.opensuse.org/devel/libraries/libyui/#{project_path}/containers/libyui-devel"
+    end
+
+    line.replace("FROM #{registry}\n")
   else
-    line.sub!(/\s*$/, docker_tag + "\n")
+    # old Docker Hub image
```

Comment:
do we still need to support it? I think SP6 won't happen, so only SP3 which already use new one, not?

Reply:
I do not know, maybe some forgotten module still uses that... I'd just keep it.